### DISCUSSION TITLE
How do I deploy a bun app without docker?

### DISCUSSION THREAD

**Discussion by jdion84** [oven-sh/bun#7952] [Q&A] [answered]
Do I just use supervisor to keep the bun server online?

**Comment by eknowles** [comment, marked as answer]
There is published [guides on using PM2](https://bun.sh/guides/ecosystem/pm2) if that helps?

**Comment by truongan07** [comment]
I did implementation ArchLinux (you can use Ubuntu, CentOS, ...) + Tmux + Nginx + Bun instead of docker and easy as PHP

**Comment by sistematico** [comment]
> Do I just use supervisor to keep the bun server online?
> 
> There's nothing in the docs about this. Bun is going nowhere if deployment isn't as easy as PHP. 

Yes, easy as PHP.

https://bun.sh/guides/ecosystem/systemd
